app/styles/newcomponents/ilios-chart-tooltip.scss

```diff
@@ -1,37 +0,0 @@
-.ilios-chart-tooltip {
```

Comment:
Tooltip styling is missing when testing.  Content does render.
<img width="551" alt="screen shot 2017-06-23 at 7 44 02 am" src="https://user-images.githubusercontent.com/597090/27487324-457663da-57e8-11e7-9208-c297a509d401.png">